Spanner/src/Instance.php

```diff
@@ -530,7 +538,8 @@ public function database($name, array $options = [])
             isset($options['sessionPool']) ? $options['sessionPool'] : null,
             $this->returnInt64AsObject,
             isset($options['database']) ? $options['database'] : [],
-            isset($options['databaseRole']) ? $options['databaseRole'] : ''
+            isset($options['databaseRole']) ? $options['databaseRole'] : '',
+            isset($options['isolationLevel']) ? $options['isolationLevel'] : $this->isolationLevel,
```

Comment:
We can use better syntax now (feel free to update the others in this list too, because this syntax has been around for many moons)
```suggestion
            $options['isolationLevel'] ?? $this->isolationLevel,
```